test/tests/postprocessors/element_average_material_property/element_average_material_property.i

```diff
@@ -0,0 +1,47 @@
+[GlobalParams]
```

Comment:
I'm going to discourage the use of GlobalParams in tests. I know it makes things more convenient but it also makes things less clear, which matters when people tend to look at these files for examples. Can you please put the desired execute flag in each relevant block?

Reply:
Will do.